pkg/chrome/chrome_linux.go

```diff
@@ -20,7 +20,7 @@ import (
 
 	"golang.org/x/crypto/pbkdf2"
 
-	"github.com/zalando/go-keyring/secret_service"
+	ss "github.com/zalando/go-keyring/secret_service"
```

Comment:
Is this necessary?

Reply:
The package is actually called that way ![grafik](https://user-images.githubusercontent.com/48837958/96903040-9789b880-1495-11eb-86c9-c82cb28afb75.png). Showing this with the alias makes it clear.

Im not a fan of that abbreviation though (nazi related)  - I'll rename it to the url-end (secret_service).